src/components/inputComponents/imageInput.js

```diff
@@ -0,0 +1,10 @@
+export default function imageInput() {
+  return (
+    <input
+      type="file"
+      id="imageInput"
```

Comment:
same here, i'd name this input `cakeImage` for instance. You also don't need to put `input` in the name of every input, it's kinda repeating it already.